### PR TITLE
feat(aws): add AWS Transfer Family to supported resources

### PR DESCRIPTION
**Pull request by vdmgolub**
The resource is introduced here: https://github.com/infracost/infracost/pull/1112

### CODE REVIEW

**Comment by alikhajeh1** [comment]
this PR should go into the v0.9.13 branch (there's an open PR for it)

**Comment by vdmgolub** [comment]
@alikhajeh1 Thanks for the heads-up! Done 👍.

File: docs/supported_resources.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ GovCloud and China regions are also supported.
 | Redshift | `aws_redshift_cluster` | |
 | Relational Database Service (RDS) | `aws_rds_cluster`, `aws_db_instance`, `aws_rds_cluster_instance` | |
 | Route 53 | `aws_route53_record`, `aws_route53_zone`, `aws_route53_resolver_endpoint`, `aws_route53_health_check` |  |
+| Transfer Family | `aws_transfer_access`, `aws_transfer_server`, `aws_transfer_ssh_key`, `aws_transfer_user` | |
```
Comment (aliscott) [inline docs/supported_resources.md:59]: We don't normally list the free resources here yet.

Comment (vdmgolub) [reply docs/supported_resources.md]: Thanks! Fixed.

**Pull request merged**